tmt/beakerlib.py

```diff
@@ -55,17 +55,20 @@ class Library(object):
     workdir or into 'destination' if provided in the identifier.
     """
 
-    def __init__(self, identifier, parent=None):
+    def __init__(self, identifier, parent=None, gathered_hlp=[]):
```

Comment:
What does `hlp` stand for?

Reply:
hlp is for helper.

Reply:
Hm, helper is not very useful to know what it is for. Looking at its purpose, what about naming it something like `gathered_library_names`. Or feel free to propose something else.